src/Toolbar/RightElement.react.js

```diff
@@ -162,7 +162,7 @@ class RightElement extends PureComponent {
             } else {
                 result.push(<IconToggle
                     key="searchIcon"
-                    name="search"
+                    name={this.props.searchable.icon ? this.props.searchable.icon : 'search'}
```

Comment:
can you please use inseted of `name={this.props.searchable.icon ? this.props.searchable.icon : 'search'}` -> `name={searchable.icon ? searchable.icon : 'search'}`